packages/scripts/Gruntfile.js

```diff
@@ -0,0 +1,62 @@
+module.exports = function( grunt ) {
+
+	'use strict';
+
+	const pkg = grunt.file.readJSON( 'package.json' );
+	const pluginSlug = grunt.option( 'pluginSlug' ) || pkg.name;
+	const textDomain = grunt.option( 'textDomain' ) || '';
+
+	console.log( pluginSlug, textDomain );
+
+	// Project configuration
+	grunt.initConfig( {
+
+		pkg,
+
+		addtextdomain: {
+			options: {
+				textdomain: textDomain,
+				updateDomains: true
+			},
+			target: {
+				files: {
+					src: [ '*.php', '**/*.php', '!\.git/**/*', '!bin/**/*', '!node_modules/**/*', '!tests/**/*' ]
+				}
+			}
+		},
+
+		wp_readme_to_markdown: {
+			your_target: {
```

Comment:
This is another area where `pkg.wordpress` could come in handy, since it would allow screenshots to be passed in.